### DISCUSSION TITLE
picking profiles

### DISCUSSION THREAD

**Discussion by beroal** [roddhjav/apparmor.d#287] [Q&A] [answered]
Several issues (#76, #77, #87) mention the command `pick` for installing profiles selectively. Can I use it, please? I don't see it in the AUR package.

**Comment by roddhjav** [comment, marked as answer]
It has been removed in favor of https://apparmor.pujol.io/install/#partial-install.

**Reply by beroal** [reply]
Do I understand it correctly that `sudo make pass` would install the profile for `pass` and I don't need to install the AUR package?

**Reply by roddhjav** [reply]
Yes, but you have to manage profile dependencies manually (and that may be a pain)

**Reply by beroal** [reply]
Not more painful than copying profiles from the Git repository, and I do this now 🙂.

So Pacman wouldn't manage profiles. So no updates.

What I imagine is a command in `PKGBUILD` that includes only the profiles listed in a configuration file in `/etc` into a package.
